client/src/components/SidePanel.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <span class='side-panel' v-if='options.length > 0'>
+    <geo-id-section
+      v-for='option in options'
+      :key='option'
+      :geoId='getGeoIdForZoomLevel(option)'
+      :geoIdInfo='getGeoIdInfoForZoomLevel(option)'
+      :selected='getSelected(option)'
+      @click='setSelected(option)' />
+  </span>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import GeoIdSection from './GeoIdSection.vue';
+import { dispatch, useSelector } from '../model/store';
+import { GeoDataState } from '../model/states/geo_data_state';
+import { MapDataState } from '../model/states/map_data_state';
+import { ZoomLevel } from '../model/states/model/map_data';
+import { GeoDataUtil } from '../util/geo_data_util';
+import { GeoData } from '../model/states/model/geo_data';
+import { setZoomLevel } from '../model/slices/map_data_slice';
+import { ScorecardMessages } from '../assets/messages/scorecard_messages';
+import { LeadDataUtil } from '../util/lead_data_util';
+import { LeadDataState } from '../model/states/lead_data_state';
+
+/**
+ * Side panel containing extra contexts for Geo IDs in the scorecard view.
+ */
+export default defineComponent({
+  name: 'SidePanel',
+  components: { GeoIdSection },
+  setup() {
+    const geoState = useSelector((state) => state.geos) as GeoDataState;
+    const mapState = useSelector((state) => state.mapData) as MapDataState;
+    const leadState = useSelector((state) => state.leadData) as LeadDataState;
+    return {
+      geoState,
+      leadState,
+      mapState,
+    };
+  },
+  data() {
+    return {
+      options: [] as ZoomLevel[],
+      selectedOption: null as ZoomLevel | null,
+      ZoomLevel,
+    };
+  },
+  methods: {
+    /**
+     * Returns the geo ID value corresponding to the given zoom level.
+     */
+    getGeoIdForZoomLevel(level: ZoomLevel): string | undefined {
```

Comment:
I think it'd be good to format the name here in a more human-friendly way. For addresses, that might mean using title case. For water systems, can we print the water system name instead of its ID?

Reply:
Makes sense - put a TODO for water system because right now we aren't returning the name so this will take an API change (we also need to make a change there to prefer a water system with results, so I'll make both changes at once). 

For parcels I added a function to format the address in all lowercase except for the state USPS code. Additionally added text transform on the div to make it sentence case -- I think this combo is the least complex of the solutions. Ideally all transformation would happen in the function but manually capitalizing each word would be more complex than the css I think - let me know if you prefer this though.